packages/definitions/dist/fhir/r4/search-parameters-medplum.json

```diff
@@ -693,6 +693,24 @@
         "type": "token",
         "expression": "Communication.topic"
       }
+    },
+    {
+      "fullUrl": "https://medplum.com/fhir/SearchParameter/Subscription-author",
+      "resource": {
+        "resourceType": "SearchParameter",
+        "id": "Subscription-author",
+        "url": "https://medplum.com/fhir/SearchParameter/Subscription-author",
+        "version": "4.0.1",
+        "name": "author",
+        "status": "draft",
+        "publisher": "Medplum",
+        "description": "The author of the Subscription resource",
+        "code": "author",
+        "base": ["Subscription"],
+        "type": "reference",
+        "expression": "Subscription.meta.author",
+        "target": ["Patient", "Practitioner", "RelatedPerson", "ClientApplication", "Bot"]
```

Comment:
Not sure if this is the right enum for all the possible `author` resource types, feedback would be appreciated

Reply:
That looks correct to me.  

The "FHIR"-y way to manage that would probably be to have a proper `CodeSystem` and `ValueSet`.  Low pri, future work.